### ISSUE TITLE
eval on mot17

### ISSUE THREAD

**Issue by YHS0201** [MCG-NJU/MeMOTR#29]
sorry to bother, I am confused about evaluation on MOT17，use the main.py and pretrained model provided, always meet the error  FileNotFoundError about val. I would like to know more detailed details of the evaluation. hope you can help me

**Comment by HELLORPG** [comment]
Since the MOT17 dataset does not provide the official validation set, our evaluation function does not support MOT17. This has also significantly hindered our explorations on MOT17, as we have been unable to find a reasonable evaluation setting to validate the effectiveness of our enhancements. 

If you need to perform evaluations on MOT17 (val), you could consider modifying the code and splitting the MOT17 train set into two parts, with one part used for training and the other for evaluation (half_train and half_val for some tracking-by-detection methods).

**Comment by YHS0201** [comment]
Thanks for help，but I'd like to know how you got results on mot17 and how I can reproduce them to get similar results

**Comment by HELLORPG** [comment]
You could refer to [submit section](https://github.com/MCG-NJU/MeMOTR?tab=readme-ov-file#submit) for MOT17. This script will help you generate the tracker files, and you can get the evaluation metrics by submitting them to the server of the MOT17 dataset.
